### ISSUE TITLE
v3.0 doesn't install into Umbraco v10

### ISSUE THREAD

**Issue by cheeseytoastie** [ArnoldV/Our.Umbraco.GMaps#164]
Umbraco v10.8.3 - added Gmaps via Nuget Package Manager. v3.0

Shows me the readme but doesn't seem to work.

Selecting v2.1.3 seems to work fine.

**Comment by AllanMT** [comment]
Yes, I found version 3.0.2 appeared to install into Umbraco 10.8.7 but the GMap editor is not available

**Comment by MarkMcDonald** [comment]
Hitting the same issue, doesn't look like the property editor is getting registered.
